### ISSUE TITLE
GPG key expired on 03/23/2024

### ISSUE THREAD

**Issue by rosaLux161** [arangodb/arangodb#20775]
## My Environment

* __Operating System__:      Debian Bookworm
* __Used Package__:            https://download.arangodb.com/arangodb311/DEBIAN/

## Component, Query & Data

__Affected feature__:

GPG key expired on 03/23/2024:

```
----------------------------------
pub   rsa4096 2015-06-02 [SC] [expired: 2024-03-23]
      CD8CB0F1E0AD5B52E93F41E7EA93F5E56E751E9B
uid           [ expired] Frank Celler (ArangoDB Debian Repository) <info@arangodb.com>
```

**Comment by BrettKorp** [comment]
This is also impacting previous versions as well. For example, I'm using the following package on Ubuntu 22: https://download.arangodb.com/arangodb34/DEBIAN/ 

Example:
```
curl -OL https://download.arangodb.com/arangodb34/DEBIAN/Release.key
gpg --show-keys --with-fingerprint Release.key
```

Outputs
```
pub   rsa4096 2015-06-02 [SC] [expired: 2024-03-23]
      CD8C B0F1 E0AD 5B52 E93F  41E7 EA93 F5E5 6E75 1E9B
uid                      Frank Celler (ArangoDB Debian Repository) <info@arangodb.com>
sub   rsa4096 2015-06-02 [E] [expired: 2024-03-23]
```

**Comment by bananasmall** [comment]
i have same problem, this is also impact for version 3.8 
`pub   rsa4096 2015-06-02 [SC] [expired: 2024-03-23]
      CD8C B0F1 E0AD 5B52 E93F  41E7 EA93 F5E5 6E75 1E9B
uid           [ expired] Frank Celler (ArangoDB Debian Repository) <info@arangodb.com>`

any information updated key there?

**Comment by dothebart** [comment]
Hi, 
please note that 3.8 is EOL, hence the binary packages are not re-pushed. 
At the time they were published, the key was still valid.

**Comment by bananasmall** [comment]
hi @dothebart hmmmm oke i wanna change it to 3.9 are this version is EOL to?

**Comment by dothebart** [comment]
yes. At the moment the last 3.10 release has been out and further maintainance stopped. 
Hence you should at least upgrade to 3.11. 
Please note that only one minor version at once can be upgraded. So the best way is: 
- upgrade to last 3.8
- upgrade to last 3.9
- upgrade to last 3.10
- upgrade to last 3.11

**Comment by bananasmall** [comment]
hi @dothebart 

i have install the latest version but still have problem
```
FATAL ERROR: EXIT_CODE_RESOLVING_FAILED for code 132 - could not resolve exit code 132
dpkg: error processing package arangodb3 (--configure):
 installed arangodb3 package post-installation script subprocess returned error exit status 132
Errors were encountered while processing:
 arangodb3
E: Sub-process /usr/bin/dpkg returned an error code (1)
``` 
i have search there is sound like the cpu support sse4 problem, i have check my server i think supp
ort sse4.1 and sse4.2

<img width="1232" alt="Screenshot 2024-05-30 at 12 01 32" src="https://github.com/arangodb/arangodb/assets/85330919/7a135cb7-c2bf-4e91-90a4-f8d2543594f1">

any suggest?
